### PR TITLE
Use builtin encoding `base64url`

### PR DESCRIPTION
**Pull request by nikeee**
Node 16+ supports base64url:
https://nodejs.org/docs/latest-v16.x/api/buffer.html

### CODE REVIEW

File: src/urlsafe-base64-helper.js [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-  base64 += Array(5 - (base64.length % 4)).join('=');
-
-  base64 = base64
-    .replace(/-/g, '+') // Convert '-' to '+'
-    .replace(/_/g, '/'); // Convert '_' to '/'
-
-  // change from urlsafe-base64 since new Buffer() is deprecated
-  return Buffer.from(base64, 'base64');
+  return Buffer.from(base64, 'base64url');
 }
```
Comment (marco-c) [inline src/urlsafe-base64-helper.js:17]: Could you just remove these functions now and use `buffer.toString` and `Buffer.from` directly in the call sites?

Comment (nikeee) [reply src/urlsafe-base64-helper.js]: Done. I left `validate` in there, because there is no equivalent for that.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 coverage/
 node_modules/
 test/output/
+.nyc_output/
```
Comment (marco-c) [inline .gitignore:4]: This is an acceptable change, but could you do it in a separate PR?
I prefer to have each PR be self-contained

Comment (nikeee) [reply .gitignore]: Done.
Maybe we should squash this when merging. Didn't have the time to do that.

**Review by marco-c — APPROVED** [review]
Thanks!
